app.test.js

```diff
@@ -3,104 +3,138 @@ const app = require("./app");
-      return request(app)
-      .get('/api/reviews')
+});
+describe("/api/reviews", () => {
+  it("GET 200: should respond with a reviews array of review objects ", () => {
+    return request(app)
+      .get("/api/reviews")
       .expect(200)
-      .then(({body}) => {
-        const {reviews} = body
+      .then(({ body }) => {
+        const { reviews } = body;
         expect(reviews).toBeInstanceOf(Array);
         expect(reviews).toHaveLength(13);
-          reviews.forEach((review) => {
-            expect(review).toMatchObject({
-              owner: expect.any(String),
-              title: expect.any(String),
-              review_id: expect.any(Number),
-              category: expect.any(String),
-              designer: expect.any(String),
-              review_img_url: expect.any(String),
-              created_at: expect.any(String),
-              votes: expect.any(Number),
-              comment_count: expect.any(Number)
-            });
+        reviews.forEach((review) => {
+          expect(review).toMatchObject({
+            owner: expect.any(String),
+            title: expect.any(String),
+            review_id: expect.any(Number),
+            category: expect.any(String),
+            designer: expect.any(String),
+            review_img_url: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            comment_count: expect.any(Number),
           });
-          expect(reviews).toBeSorted({ descending: true });
-      })
-
-    });
+        });
+        expect(reviews).toBeSorted({ descending: true });
+      });
   });
-
-
+});
+describe("/api/reviews/:review_id/comments", () => {
+  it("GET 200: should respond with an array of comments for the given review_id of each comment", () => {
+    return request(app)
+      .get("/api/reviews/3/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toBeInstanceOf(Array);
+        comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            author: expect.any(String),
+            body: expect.any(String),
+            review_id: 3,
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+          });
+        });
+        expect(comments).toBeSorted({ descending: true });
+      });
+  });
+  it("400: should respond with an error message indicating requested id is invalid. ", () => {
+    return request(app)
+      .get("/api/reviews/not-a-num/comments")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad Request");
+      });
+  });
+  it("404: should respond with correct msg for valid but non-existent review_id.", () => {
+    return request(app)
+      .get("/api/reviews/5/comments")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("comments not found!");
+      });
+  });
```

Comment:
if the review is real it should send back an empty array if no comments, so should be a 404

the 404 should be for a non-existant id :)